packages/server/src/fhir/references.ts

```diff
@@ -11,7 +11,7 @@ import {
 } from '@medplum/core';
 import { OperationOutcomeIssue, Reference, Resource } from '@medplum/fhirtypes';
 import { randomUUID } from 'crypto';
-import { getSystemRepo, Repository } from './repo';
+import { Repository, getSystemRepo } from './repo';
```

Comment:
We need to align our import sorting config 😔 

Normally I ignore these, but given that this is the only change in the file, I'd recommend we revert.